resources/qml/TimelineRow.qml

```diff
@@ -71,27 +71,39 @@ Item {
         gesturePolicy: TapHandler.ReleaseWithinBounds
     }
 
-    RowLayout {
+    Item {
         id: row
 
         anchors.rightMargin: 1
```

Comment:
Those 1px margins might not be needed anymore. They were a workaround for hover handling not working otherwise, but that might have been fixed by the qml root. I need to test that though.